src/main/java/org/upf/gestion_quiz/gestion_quiz_enligne/dao/ReponseDAO.java

```diff
@@ -0,0 +1,80 @@
+package org.upf.gestion_quiz.gestion_quiz_enligne.dao;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.EntityTransaction;
+import javax.persistence.Persistence;
+import javax.persistence.Query;
+
+import org.upf.gestion_quiz.gestion_quiz_enligne.Entity.Reponse;
+import org.upf.gestion_quiz.gestion_quiz_enligne.interf.IntGlobale;
+
+public class ReponseDAO implements IntGlobale<Reponse> {
+	
+	EntityManagerFactory emf = Persistence.createEntityManagerFactory("gestion_quiz_enligne");
+	EntityManager em 		 = emf.createEntityManager();
+	EntityTransaction tx     = em.getTransaction();
+
+	@Override
+	public boolean Ajouter(Reponse obj) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.